src/include/platform/DeviceInfoProvider.h

```diff
@@ -131,6 +131,14 @@ class DeviceInfoProvider
      */
     CHIP_ERROR AppendUserLabel(EndpointId endpoint, const UserLabelType & label);
 
+    uint16_t GetSimultaneousInvocationsSupported();
+
+    uint16_t GetSimultaneousWritesSupported();
+
+    uint16_t GetReadPathsSupported();
+
+    uint16_t GetSubscribePathsSupported();
+
```

Comment:
Please:
- Make 1 new virtual method that returns a struct, that way, it can be overridden.